cheatsheets/XML_External_Entity_Prevention_Cheat_Sheet.md

```diff
@@ -549,6 +549,24 @@ libxml_disable_entity_loader(true);
 
 A description of how to abuse this in PHP is presented in a good [SensePost article](https://www.sensepost.com/blog/2014/revisting-xxe-and-abusing-protocols/) describing a cool PHP based XXE vulnerability that was fixed in Facebook.
 
+# PYTHON
+
+The following information for XXE injection in PYTHON 3 is directly from this [official PYTHON 3 documentation page](https://docs.python.org/3/library/xml.html#xml-vulnerabilities).
```

Comment:
I'd probably write `the official` (this applies below as well).

Reply:
The link text should really indicate that it relates to `xml`. (People shouldn't have to read the url which isn't visible to figure this out.)